src/components/AvatarWrapper/AvatarWrapper.vue

```diff
@@ -167,7 +167,7 @@ export default {
 			case ATTENDEE.ACTOR_TYPE.PHONES:
 				return 'icon-phone'
 			case ATTENDEE.ACTOR_TYPE.BOTS:
-				return this.id === ATTENDEE.CHANGELOG_BOT_ID ? 'icon-changelog' : ''
+				return this.id === ATTENDEE.CHANGELOG_BOT_ID || this.id === ATTENDEE.SAMPLE_BOT_ID ? 'icon-changelog' : ''
```

Comment:
For readability. Not sure that's expected to work correctly without parenthesis
```suggestion
				return [ATTENDEE.CHANGELOG_BOT_ID, ATTENDEE.SAMPLE_BOT_ID].includes(this.id) ? 'icon-changelog' : ''
```

Reply:
![grafik](https://github.com/user-attachments/assets/e72b90d0-cd46-4a17-95c7-e0d229ab3945)

Basic boolean logic :see_no_evil:  But okay, changed it